### PR TITLE
Don't initialize the ZIP asset bundle as a directory.

### PR DESCRIPTION
**Pull request by chinmaygarde**
Regression introduced in https://github.com/flutter/engine/commit/cee7fef7

### CODE REVIEW

**Comment by chinmaygarde** [comment]
Fixes https://github.com/flutter/flutter/issues/5395

cc @Hixie

File: sky/shell/ui/engine.h [hunk 1]
```diff
@@ -92,8 +92,6 @@ class Engine : public UIDelegate,
   void RunFromSnapshotStream(const std::string& script_uri,
                              mojo::ScopedDataPipeConsumerHandle snapshot);
 
-  void SetupAssetBundle(const mojo::String& bundle_path);
```
Comment (Hixie) [inline sky/shell/ui/engine.h]: did you mean to remove this?

**Comment by Hixie** [comment]
RSLGTM

File: sky/shell/ui/engine.h (hunk 1 above)
Comment (chinmaygarde) [inline sky/shell/ui/engine.h]: Yeah, there is no corresponding implementation.

File: sky/shell/ui/engine.cc [hunk 2]
```diff
@@ -90,7 +90,7 @@ void Engine::RunFromSource(const std::string& main,
   if (packages_path.empty())
     packages_path = FindPackagesPath(main);
   if (!bundle.empty())
-    ConfigureDirectoryAssetBundle(bundle);
+    ConfigureZipAssetBundle(bundle);
```
Comment (johnmccutchan) [inline sky/shell/ui/engine.cc]: This will break assets in hot mode.

**Pull request merged**